src/model/add-dots.ts

```diff
@@ -0,0 +1,60 @@
+// Check if the block contains a dot, replace with an empty span with a bullet attribute
+const hasDots = (element: CAPIElement): boolean => {
+	if (element._type !== 'model.dotcomrendering.pageElements.TextBlockElement')
+		return false;
+
+	if (element.html.includes('•')) {
+		element.html = element.html.replace(
+			new RegExp('[•]', 'g'),
+			'<span data-dcr-style="bullet"></span>',
+		);
```

Comment:
This _might_ work but string manipulation on html using regexes is _potentially_ brittle. For example, this doesn't deal with the html tags that might be wrapped around the dot. Eg. By using spans and inserting them inside the containing elements, we risk style conflicts where the browser might try to apply styles for a `em` or `li` tag but the span competes?